php/send-http-request/index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+require 'vendor/autoload.php';
+
+use GuzzleHttp\Client;
+
+return function($req, $res) {
+    try {
+        $payload = \json_decode($req['payload'], true);
+        $url = \trim($payload['url']);
+        $method = \trim($payload['method']);
+        $body = \trim($payload['body']);
+        $headers = $payload['headers'];
```

Comment:
body and headers are optional and not passing them will result in warnings:

```
Warning: Undefined array key "body" in /usr/code-start/index.php on line 12

Deprecated: trim(): Passing null to parameter #1 ($string) of type string is deprecated in /usr/code-start/index.php on line 12

Warning: Undefined array key "headers" in /usr/code-start/index.php on line 13
```

Could you please do some checking to avoid the warnings?